usr/share/rear/finalize/Debian/i386/550_rebuild_initramfs.sh

```diff
@@ -1,16 +1,24 @@
 
-# Rebuild the initramfs if the drivers changed:
+# Rebuild the initramfs:
 
-# Skip if there is nothing to do.
-# During "rear recover" 260_recovery_storage_drivers.sh creates $TMP_DIR/storage_drivers
-if ! test -s $TMP_DIR/storage_drivers ; then
-    Log "Skip recreating initramfs: No needed storage drivers ('$TMP_DIR/storage_drivers' is empty)"
+# Skip if it is explicitly wanted to not rebuild the initramfs:
+if is_false $REBUILD_INITRAMFS ; then
+    Log "Skip recreating initramfs (REBUILD_INITRAMFS is false)"
     return 0
 fi
-# During "rear mkbackup/mkrescue" 260_storage_drivers.sh creates $VAR_DIR/recovery/storage_drivers
-if cmp -s $TMP_DIR/storage_drivers $VAR_DIR/recovery/storage_drivers ; then
-    Log "Skip recreating initramfs: '$TMP_DIR/storage_drivers' and '$VAR_DIR/recovery/storage_drivers' are the same"
-    return 0
+
+# Skip if not needed but only when it is not explicitly wanted to rebuild the initramfs in any case:
+if ! is_true $REBUILD_INITRAMFS ; then
```

Comment:
@gdha 
see
https://github.com/rear/rear/pull/1309#issuecomment-294789330
and
https://github.com/rear/rear/pull/1212#issuecomment-283698753
and starting at
https://github.com/rear/rear/blob/master/usr/share/rear/lib/global-functions.sh#L24
and
https://github.com/rear/rear/wiki/Coding-Style
<pre>
Relax-and-Recover functions
...
Note that ! is_true is not the same as is_false.
</pre>
;-)